test/k8sT/Services.go

```diff
@@ -2056,6 +2059,15 @@ Secondary Interface %s :: IPv4: (%s, %s), IPv6: (%s, %s)`, helpers.DualStackSupp
 			})
 		})
 
+		// SA needs 4.19. However, on the 4.19 job we run KPR. So to test SA on
+		// bpf_lxc with kube-proxy we need to disable KRP.
+		SkipItIf(helpers.DoesNotRunOn419Kernel, "Tests kube-proxy ClusterIP with sessionAffinity from pods", func() {
+			DeployCiliumOptionsAndDNS(kubectl, ciliumFilename, map[string]string{
+				"kubeProxyReplacement": "disabled",
+			})
```

Comment:
We then need to redeploy Cilium with KPR at the end of the test or the next test (`Tests IPv6 NodePort Services`) will run with kube-proxy handling requests instead of KPR.